spyderlib/config.py

```diff
@@ -512,17 +514,45 @@ def is_ubuntu():
               'editor/toggle comment': "Ctrl+1",
               'editor/blockcomment': "Ctrl+4",
               'editor/unblockcomment': "Ctrl+5",
+              'editor/start of line': "Meta+A",
+              'editor/end of line': "Meta+E",
+              'editor/previous line': "Meta+P",
+              'editor/next line': "Meta+N",
+              'editor/previous char': "Meta+B",
+              'editor/next char': "Meta+F",
+              'editor/previous word': "Meta+Left",
+              'editor/next word': "Meta+Right",
+              'editor/kill to line end': "Meta+K",
+              'editor/kill to line start': "Meta+U",

```

Comment:
Wow, Unity is terrible.  I tried using Alt+Key shortcuts in both Spyder and Sublime, and neither are registered.  Using `Ctrl+Shift+Key` worked, but I can't see using that as the default just because of one DE.  These are not major shortcuts, and could always be remapped by a power user who happened to be using Unity.